src/components/Shifts/ShiftDetails.jsx

```diff
@@ -0,0 +1,173 @@
+import React, { useState } from 'react';
+// import { useParams } from "react-router-dom";
```

Comment:
```suggestion
```

Reply:
I left this in because it's part of the other commented out code that will make this work once we have redux set up. I can still remove it if you want me to tho.

Reply:
Just leave it then